redbot/cogs/audio/core/commands/controller.py

```diff
@@ -617,8 +617,14 @@ async def command_stop(self, ctx: commands.Context):
     @commands.guild_only()
     @commands.cooldown(1, 15, commands.BucketType.guild)
     @commands.bot_has_permissions(embed_links=True)
-    async def command_summon(self, ctx: commands.Context):
-        """Summon the bot to a voice channel."""
+    async def command_summon(
+        self, ctx: commands.Context, *, voice_channel: Optional[discord.VoiceChannel] = None
+    ):
+        """Summon the bot to a voice channel.
+
+        You can specify a personalised voice room with an id. Otherwise, the bot will join the voice room you are in.
+        To learn how to get a channel ID please read the following help article: <https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID>
```

Comment:
```suggestion
        To learn how to get a channel ID please read [this help article](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID).
```
In this case, it might look more presentable to use hyperlink syntax instead of just having the link as it is (see my suggestion)

Reply:
The one issue with hyperlinks is that in non-embed help it renders without formatting. I can't remember if we've accepted this limitation in other commands with links in core, but if we have not I'd say to continue to not use them.